geniza/corpus/admin.py

```diff
@@ -161,6 +166,46 @@ def save_model(self, request, obj, form, change):
             obj.last_modified = None
         super().save_model(request, obj, form, change)
 
+    ## CSV EXPORT -------------------------------------------------------------
+
+    export_fields = [
+        'id', 'shelfmark', 'description', 'doctype',
+        'all_tags', 'all_languages', 'all_probable_languages',
+        'last_modified', 'notes', 'needs_review', 'status'
+    ]
+
+    def csv_filename(self):
+        '''Generate filename for CSV download'''
+        return f'geniza-documents-{now().strftime("%Y%m%dT%H%M%S")}.csv'
```

Comment:
maybe should use pgp instead of geniza here (more specific)

Reply:
In a recent conversation, you preferred geniza. I'm happy to change this, but would you mind confirming?

Reply:
ha! I don't remember that. this seems fine!